tools/isopack.js

```diff
@@ -1117,7 +1117,8 @@ _.extend(Isopack.prototype, {
           // If we're going to write a legacy prelink file later, track the
           // original form of the resource object (with the source in a Buffer,
           // etc) instead of the later version.  #HardcodeJs
-          if (writeLegacyBuilds && resource.type === "source" &&
+          if (writeLegacyBuilds &&
```

Comment:
We could just put something on the resource json thing when we generate it
that says that it is (or isn't) from the default handler...